### PR TITLE
Add Sysdig image scanner integration

### PR DESCRIPTION
**Pull request by airadier**
Adding sysdig-image-scanner integration to the integrations.yaml.

Let me know if the logo size and format is ok or any change is required.

Thanks very much!

### CODE REVIEW

File: docs/website/data/integrations.yaml [hunk 1]
```diff
@@ -690,6 +690,20 @@ integrations:
         link: https://www.youtube.com/watch?v=jrrW855xL3s
 
 
+  sysdig-image-scanner:
+    title: Sysdig Image Scanner admission controller
```
Comment (timothyhinrichs) [inline docs/website/data/integrations.yaml:694]: Add "Kubernetes" to the title

File: docs/website/data/integrations.yaml [hunk 2]
```diff
@@ -690,6 +690,20 @@ integrations:
...
+    description: Sysdig’s OPA Image Scanner combines Sysdig Secure image image scanner with OPA policy-based rego language to evaluate the scan results and the admission context, providing great flexibility on the admission decision.
```
Comment (timothyhinrichs) [inline docs/website/data/integrations.yaml:695]: Remove duplicated "image"

File: docs/website/data/integrations.yaml (hunk 1 above)
Comment (airadier) [reply docs/website/data/integrations.yaml]: Done! Thanks

File: docs/website/data/integrations.yaml (hunk 2 above)
Comment (airadier) [reply docs/website/data/integrations.yaml]: Done! Thanks

File: docs/website/data/integrations.yaml [hunk 3]
```diff
@@ -690,6 +690,20 @@ integrations:
         link: https://www.youtube.com/watch?v=jrrW855xL3s
 
 
+  sysdig-image-scanner:
+    title: Kubernetes Sysdig Image Scanner admission controller
```
Comment (patrick-east) [inline docs/website/data/integrations.yaml:694]: nit: Can you make this "title case", eg `Kubernetes Sysdig Image Scanner Admission Controller`

Comment (airadier) [reply docs/website/data/integrations.yaml]: Sure, done!

**Comment by airadier** [comment]
> LGTM 👍
> 
> Please squash the commits and then we can get this merged.

Thanks, it is squashed now.

Regards

**Pull request merged**